eva-accession-clustering/src/test/resources/test-data/monotonic_accession_recovery_agent_category_rs_test_data.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO contiguous_id_blocks
+VALUES (1, 'test-instance-recover-state-00', 'rs', 3000000000, 2999999999, 3000000029, true, '1970-01-01 00:00:00');
+
+INSERT INTO contiguous_id_blocks
+VALUES (2, 'test-instance-recover-state-00', 'rs', 3000000030, 3000000029, 3000000059, true, '1970-01-01 00:00:00');
+
+INSERT INTO contiguous_id_blocks
+VALUES (3, 'test-instance-recover-state-00', 'rs', 3000000060, 3000000059, 3000000089, true, '1970-01-01 00:00:00');
+
+INSERT INTO contiguous_id_blocks
+VALUES (4, 'test-instance-recover-state-00', 'rs', 3000000090, 3000000089, 3000000119, true, '1970-01-01 00:00:00');
+
+INSERT INTO contiguous_id_blocks
+VALUES (5, 'test-instance-recover-state-00', 'rs', 3000000120, 3000000119, 3000000149, true, '2099-01-01 00:00:00');
```

Comment:
Can't wait for this test to fail in 2100...

(Just kidding, if the code we write lasts that long I'll be impressed...)